### PR TITLE
fillToFit option for depiction

### PR DESCRIPTION
**Pull request by rickhelmus**
Hello,

This PR adds support to use the `fillToFit` option from CDK depiction, which allows plotted structures to maximize their size.

Thanks,
Rick

### CODE REVIEW

**Review by zachcp — APPROVED** [review]
@rickhelmus  I am late to reviewing. My main objection is that you've deleted the Jar files. If you can modify your PR today I'll pull this in.

**Comment by rickhelmus** [comment]
Hi @zachcp ,

Thanks for checking my PR!

Hmm I don't see any deleted Jars in the changed files list. Which files do you mean specifically?

Thanks,
Rick

File: rcdk/inst/cont/rcdk.jar
Comment (zachcp) [inline rcdk/inst/cont/rcdk.jar:1]: Can you remove this?

File: rcdkjar/rcdk.jar
Comment (zachcp) [inline rcdkjar/rcdk.jar:1]: Can you remove this?

**Comment by zachcp** [comment]
Apologies; I misspoke. Can you remove theJAR files I flagged?  I will rebuild them on my end for the next release.

**Comment by rickhelmus** [comment]
Done! ;-)

**Comment by zachcp** [comment]
Thanks! Aiming to rebuild/submit a small release today. May take a bit to work through CRAN

**Pull request merged**